inc/plugins/class-dynamic-content.php

```diff
@@ -649,6 +663,53 @@ public function get_link( $data ) {
 		return apply_filters( 'otter_blocks_evaluate_dynamic_content_link', '', $data );
 	}
 
+	/**
+	 * Mark the exceptions for Dynamic request.
+	 *
+	 * @param array $data Dynamic request.
+	 * @return array Dynamic request with marked exceptions.
+	 */
+	public function mark_exceptions( $data ) {
```

Comment:
NIT, it might be more relevant to keep this code in Neve, i.e. have some filter and hook this exception in Neve since it's compatibility for Otter vs. the way around.

Reply:
Yea, I agree with both in this, but I do expect that this problem can happen in just any plugin that does this, so it's more of an Otter problem than Neve.